juniper_official/routing/collect-fib-stats.rule

```diff
@@ -0,0 +1,127 @@
+/*
+ * Collects FIB statistics for plotting and ML
+ */
+ healthbot {
+    topic routing.fib {
+        rule collect-fib-stats {
```

Comment:
There is no triggers here. Do we require this rule?

Reply:
Updated trigger.